build/gulpfile.vscode.web.js

```diff
@@ -94,7 +94,7 @@ const createVSCodeWebProductConfigurationPatcher = (product) => {
 				commit,
 				date: buildDate
 			});
-			return content.replace('/*BUILD->INSERT_PRODUCT_CONFIGURATION*/', productConfiguration.substr(1, productConfiguration.length - 2) /* without { and }*/);
```

Comment:
These are different, `"{ab}".substr(1,"{ab}".length - 2) === "ab"` , `"{ab}".substring(1,"{ab}".length - 2) === "a"`.

Reply:
Shall I revert to .substr instead?